P5/Source/Specs/unicodeProp.xml

```diff
@@ -0,0 +1,276 @@
+                <valItem ident="scf"/>
+                <valItem ident="Script"/>
+                <valItem ident="Script_Extensions"/>
+                <valItem ident="scx"/>
+                <valItem ident="SD"/>
+                <valItem ident="Sentence_Break"/>
+                <valItem ident="Sentence_Terminal"/>
+                <valItem ident="Simple_Case_Folding"/>
+                <valItem ident="Simple_Lowercase_Mapping"/>
+                <valItem ident="Simple_Titlecase_Mapping"/>
+                <valItem ident="Simple_Uppercase_Mapping"/>
+                <valItem ident="slc"/>
+                <valItem ident="Soft_Dotted"/>
+                <valItem ident="stc"/>
+                <valItem ident="STerm"/>
+                <valItem ident="suc"/>
+                <valItem ident="tc"/>
+                <valItem ident="Term"/>
+                <valItem ident="Terminal_Punctuation"/>
+                <valItem ident="Titlecase_Mapping"/>
+                <valItem ident="uc"/>
+                <valItem ident="UIdeo"/>
+                <valItem ident="Unicode_1_Name"/>
+                <valItem ident="Unified_Ideograph"/>
+                <valItem ident="Upper"/>
+                <valItem ident="Uppercase"/>
+                <valItem ident="Uppercase_Mapping"/>
+                <valItem ident="Variation_Selector"/>
+                <valItem ident="Vertical_Orientation"/>
+                <valItem ident="vo"/>
+                <valItem ident="VS"/>
+                <valItem ident="WB"/>
+                <valItem ident="White_Space"/>
+                <valItem ident="Word_Break"/>
+                <valItem ident="WSpace"/>
+                <valItem ident="XID_Continue"/>
+                <valItem ident="XID_Start"/>
+                <valItem ident="XIDC"/>
+                <valItem ident="XIDS"/>
+                <valItem ident="XO_NFC"/>
+                <valItem ident="XO_NFD"/>
+                <valItem ident="XO_NFKC"/>
+                <valItem ident="XO_NFKD"/>
+            </valList>
+        </attDef>
+        <attDef ident="value" usage="req" mode="replace">
+            <desc versionDate="2020-01-28" xml:lang="en">specifies the  value of a named Unicode property.</desc>
+            <datatype>
+                <dataRef key="teidata.text"/>
+            </datatype>
+        </attDef>
+    </attList>
+    <exemplum versionDate="2019-07-01" xml:lang="und">
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <char xml:id="U4EBA_circled">
+              <unicodeProp name="Decomposition_Mapping" value="circle" version="12.1"/>
+              <localProp name="name" value="CIRCLED IDEOGRAPH 4EBA"/>
+              <localProp name="daikanwa" value="36"/>
+              <mapping type="standard">人</mapping>
+            </char>
+        </egXML>
+    </exemplum>
+    <remarks versionDate="2005-01-14" xml:lang="en">
+        <p>A definitive list of current Unicode property names is provided in The Unicode Standard.</p>
```

Comment:
Should have a bibliographic citation, no?

Reply:
The thing is there are different ways of getting the list. I don't think this is a showstopper but will keep it in mind if we can optimize it, without having to constantly edit the bibliography section with each commit.

Reply:
There may be different ways of getting the list, but this passage suggests a particular way. And even if we don’t point to a particular list that would need to be updated frequently, just a citation for the reference to a work is a good idea. Turns out there is already an entry for Unicode 5.0 in the bibliography. It is referred to only once, from the “Is Your Journey Really Necessary?” section of WD:

> `Alternatively, users can check the latest published version of <title>The Unicode Standard</title> (<ref target="#CH-BIBL-3">Unicode Consortium (2006)</ref>), though the web site is often more up to date than the printed version, and should be checked for preference.`

I’m thinking the reference there, and here, should be to `<title ref="#CH-BIBL-3">The Unicode Standard</title>`, without any parenthetical; and then the entry at CH-BIBL-3 should be updated to the current version.

Neither of these has to happen urgently, especially since the Stylesheets don’t (yet) do anything with that `@ref` attribute.

All that said, we are dancing around the fact we have been completely inconsistent in how we refer to websites and references and this list and stuff like it. Sigh.

Reply:
I went with a stable link to the latest version of unicode, which is accurate as of now, and will be continue to be with phase 5.